iModelCore/ECDb/ECDb/SchemaValidator.cpp

```diff
@@ -102,6 +102,8 @@ bool SchemaValidator::ValidBaseClassesRule::Validate(SchemaImportContext const&
             {
             if (Enum::Contains(ctx.GetOptions(), SchemaManager::SchemaImportOptions::DoNotFailSchemaValidationForLegacyIssues))
                 {
+                issueReporter.ReportV(IssueSeverity::Info, IssueCategory::BusinessProperties, IssueType::ECDbIssue, ECDbIssueId::ECDb_0631, 
+                "Ignoring validation error: ECClass '%s' has multiple base classes. Order is important, only the first base class can be an entity, additional ones must use mixins instead. Error suppressed as DoNotFailSchemaValidationForLegacyIssues flag is set.", ecClass.GetFullName());

```

Comment:
Let's avoid mentioning the flag name here, we know why it is being raise and it would only confuse an end user because it is not a flag they can control

Reply:
Also, there is no need to log in addition to the issue reporter because the issue report will be logged.

Reply:
Done !